presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreUtil.java

```diff
@@ -1026,6 +1027,11 @@ public static boolean isIcebergTable(Map<String, String> tableParameters)
         return ICEBERG_TABLE_TYPE_VALUE.equalsIgnoreCase(tableParameters.get(ICEBERG_TABLE_TYPE_NAME));
     }
 
+    public static boolean isIcebergView(Table table)
+    {
+        return "true".equalsIgnoreCase(table.getParameters().get(ICEBERG_VIEW_TYPE_NAME));
+    }
```

Comment:
Let's use the already existed constant `PRESTO_VIEW_FLAG`, no need to declare another one.